SEP-0012.md

```diff
@@ -0,0 +1,495 @@
+# SEP-0012 -- NDCube 2
+
+| SEP           | num                                                     |
+|---------------|---------------------------------------------------------|
+| title         | NDCube 2                                                |
+| author(s)     | [Stuart Mumford](https://orcid.org/0000-0003-4217-4642) |
+| contact email | stuart@cadair.com                                       |
+| date-creation | 2020-10-13                                              |
+| type          | standard                                                |
+| discussion    |                                                         |
+| status        | discussion                                              |
+
+# Introduction
+
+This SEP proposes a redesigned API for the next major version of the ndcube package.
+The primary objective of this SEP is to gather as much community feedback on the proposed API before it is released.
+This is also a prelude to a future change, long discussed, to have the `GenericMap` class in sunpy core inherit from `NDCube`.
+This SEP should therefore also be considered as a discussion of future changes to the `Map` API, as the API proposed here would eventually (with long deprecation) be adopted by map for the functionality provided here.
+That being said, details of the migration of Map to this API are out of scope for this SEP, just a discussion of the eventual API to be adopted.
+
+## Background
+
+The creation of ndcube was originally motivated by the desire to represent and interrogate multi-dimensional solar datasets, such as those collected with the IRIS satellite.
```

Comment:
this sentence assumes readers know what IRIS is. This whole section needs more information to onboard people that are not familiar with NDCube.

Reply:
Agreed; I think replacing "IRIS" with a concrete description of such data would be better here. Suggestion below:
```suggestion
The creation of ndcube was originally motivated by the desire to represent and interrogate solar datasets with more than two dimensions, for example two spatial and one spectral (wavelength) dimension.
```